pkg/issuer/acme/dns/clouddns/clouddns.go

```diff
@@ -19,7 +19,7 @@ import (
 
 	"github.com/go-logr/logr"
 
-	"golang.org/x/net/context"
+	"context"
```

Comment:
I think this bit is why the goimports test is failing- could you move this import to the top section together with the other standard library imports? Same in the test file.